.coffeelintignore

```diff
@@ -1,2 +0,0 @@
-iconsets
-node_modules
```

Comment:
Never mind, it's only used when running linting locally. Linting by CI services only run `lib` & `specs` folder: https://github.com/atom/ci/blob/master/build-package.sh#L75-L86.